src/main/java/org/testeditor/fixture/web/WebDriverFixture.java

```diff
@@ -881,7 +882,7 @@ public void selectElementInSelection(String elementLocator, LocatorStrategy loca
         WebElement element = getWebElement(elementLocator, locatorType);
         try {
             new Select(element).selectByVisibleText(value);
-            logger.info("Selected value: {}" , value);
+            logger.debug("Selected value: {}" , value);
```

Comment:
trace, put a debug message on top "select element {} in selection {} located by {}"